Dockerfile

```diff
@@ -33,6 +33,7 @@ RUN apt-get update && \
 	    python3-pip \
 	    apt-transport-https \
         xclip \
+	zip \
```

Comment:
can you line up the indentation for these packages?

Reply:
Sure! 

Sorry, I made it in the web editor and I didn't realize about it. 

It is done!